config/initializers/devise.rb

```diff
@@ -1,6 +1,12 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = ENV["GRANTZILLA_DEVISE_MAILER_SENDER"] || "no-reply@example.com"
```

Comment:
See commit comment here: https://github.com/on-site/grantzilla/commit/dba5474e9adaf98320151fa5237b0fac8af59b04

I don't believe there's any reason to override ActionMailer::Base.default_options[:from] here